agent/consul/server_test.go

```diff
@@ -183,10 +185,6 @@ func newServer(c *Config) (*Server, error) {
 			oldNotify()
 		}
 	}
-	// Restore old notify to guard against re-closing `up` on a retry
```

Comment:
I think you restored this yesterday, did you want to remove it again? Otherwise you might need to rebase.

Reply:
I want to remove it, don't need it anymore.